src/SleetLib/Commands/PushCommand.cs

```diff
@@ -184,6 +184,12 @@ private static async Task PushPackages(List<PackageInput> packageInputs, SleetCo
 
                 if (exists)
                 {
+                    if (await SleetUtility.IsPackageIdenticalOnFeedAsync(package.PackagePath, packageIndex, context.Source, new FlatContainer(context), log) != true)
```

Comment:
Should this be == false?  Below you can return null if the package doesn't exist in the index.

Reply:
Yeap. You're correct.

Reply:
Done